python/paddle/fluid/layers/control_flow.py

```diff
@@ -295,11 +295,13 @@ class StaticRNN(object):
     """
     StaticRNN class.
 
-    The StaticRNN can process a batch of sequence data. The length of each
-    sample sequence must be equal. The StaticRNN will have its own parameters
-    like inputs, outputs, memories. **Note that the first dimension of inputs
-    represents sequence length, and all the sequence length of inputs must be
-    the same. And the meaning of each axis of input and output are the same.**
+    The StaticRNN can process a batch of sequence data. The first dimension of inputs
+    represents sequence length, the length of each input sequence must be equal.
+    StaticRNN will unfold sequence into time steps, user need to define how to process
```

Comment:
A tiny issue. "user need to" should be changed to "user needs to" or "users need to" because "need" is the primary verb here. (When "need" is an auxiliary verb, we can say "user need define xxx", but infinitive "need to define" has to be a third person form here)